### PR TITLE
[plg_editors_codemirror]XML code style

### PR DESCRIPTION
**Pull request by wojsmol**
### Summary of Changes

Code style changes for plg_editors_codemirror
### Testing Instructions

code review

### CODE REVIEW

File: plugins/editors/codemirror/codemirror.xml [hunk 1]
```diff
@@ -177,72 +203,112 @@
...
-					<option value="Alt">PLG_CODEMIRROR_FIELD_VALUE_FULLSCREEN_MOD_ALT</option>
+				<field
+					name="fullScreenMod"
+					type="checkboxes"
+					label="PLG_CODEMIRROR_FIELD_FULLSCREEN_MOD_LABEL"
+					description="PLG_CODEMIRROR_FIELD_FULLSCREEN_MOD_DESC"
+					filter="options"
+					>
+					<option value="Shift">
+						PLG_CODEMIRROR_FIELD_VALUE_FULLSCREEN_MOD_SHIFT
```
Comment (andrepereiradasilva) [inline plugins/editors/codemirror/codemirror.xml]: still think the option tags should be in one line leike they were. Is easier to read.

File: plugins/editors/codemirror/codemirror.xml [hunk 2]
```diff
@@ -256,16 +322,24 @@
...
+					class="btn-group btn-group-yesno"
+					default="native"
+					filter="options"
 					>
-					<option value="native">PLG_CODEMIRROR_FIELD_VALUE_SCROLLBARSTYLE_DEFAULT</option>
-					<option value="simple">PLG_CODEMIRROR_FIELD_VALUE_SCROLLBARSTYLE_SIMPLE</option>
-					<option value="overlay">PLG_CODEMIRROR_FIELD_VALUE_SCROLLBARSTYLE_OVERLAY</option>
+					<option value="native">
+						PLG_CODEMIRROR_FIELD_VALUE_SCROLLBARSTYLE_DEFAULT
+					</option>
```
Comment (andrepereiradasilva) [inline plugins/editors/codemirror/codemirror.xml]: still think the option tags should be in one line like they were. Is easier to read.

Comment (wojsmol) [inline plugins/editors/codemirror/codemirror.xml]: See http://joomla.github.io/coding-standards/?coding-standards/chapters/xml.md
```
<option value="0">
        SOMETHING_VERY_LONG
</option>
```

File: plugins/editors/codemirror/codemirror.xml (hunk 1 above)
Comment (wojsmol) [inline plugins/editors/codemirror/codemirror.xml]: See http://joomla.github.io/coding-standards/?coding-standards/chapters/xml.md
```
<option value="0">
        SOMETHING_VERY_LONG
</option>
```

File: plugins/editors/codemirror/codemirror.xml (hunk 2 above)
Comment (andrepereiradasilva) [inline plugins/editors/codemirror/codemirror.xml]: yes, i know, it's not clear.
seems we disagree with the definition of "very long". IMHO very long is 100+ chars. (remember in the php code style a long line is 150+ chars.)

Comment (andrepereiradasilva) [inline plugins/editors/codemirror/codemirror.xml]: @brianteeman what do you think?

Comment (brianteeman) [inline plugins/editors/codemirror/codemirror.xml]: I agree with you Andre but when it comes to this sort of thing I don't
really have a clue

Comment (wojsmol) [inline plugins/editors/codemirror/codemirror.xml]: IMHO we need a clear definition of a long line in the XML.

Comment (andrepereiradasilva) [inline plugins/editors/codemirror/codemirror.xml]: @wojsmol plese propose a change to make this clear.
https://github.com/joomla/coding-standards/blob/gh-pages/manual/en-US/coding-standards/chapters/xml.md

Comment (Bakual) [inline plugins/editors/codemirror/codemirror.xml]: I'd go with the same length we have for PHP (150), but when I wrote the rule I didn't want to specify it on purpose. I think it depends a lot on what is around it. Use common sense.
In this case here I would leave it on one line for sure as it isn't much longer than the lines above and it's easier to read when on one line.

File: plugins/editors/codemirror/codemirror.xml [hunk 3]
```diff
@@ -25,144 +25,170 @@
...
-					   class="btn-group btn-group-yesno"
-					   default="0"
-					   filter="options"
-					   description="PLG_CODEMIRROR_FIELD_SELECTIONMATCHES_DESC"
-					   label="PLG_CODEMIRROR_FIELD_SELECTIONMATCHES_LABEL"
-				        >
+				<field
+					name="selectionMatches"
+					type="radio"
+					class="btn-group btn-group-yesno"
```
Comment (JoomliC) [inline plugins/editors/codemirror/codemirror.xml]: Code standards, order for the 4 first attributes : <code>name</code>, <code>type</code>, <code>label</code> and <code>description</code> ;-)

File: plugins/editors/codemirror/codemirror.xml [hunk 4]
```diff
@@ -25,144 +25,170 @@
...
-					   label="PLG_CODEMIRROR_FIELD_AUTOCLOSEBRACKET_LABEL"
-						>
+				<field
+					name="autoCloseBrackets"
+					type="radio"
+					class="btn-group btn-group-yesno"
+					default="1"
+					filter="options"
+					description="PLG_CODEMIRROR_FIELD_AUTOCLOSEBRACKET_DESC"
+					label="PLG_CODEMIRROR_FIELD_AUTOCLOSEBRACKET_LABEL"
```
Comment (JoomliC) [inline plugins/editors/codemirror/codemirror.xml]: Code standards, order for the 4 first attributes : <code>name</code>, <code>type</code>, <code>label</code> and <code>description</code> ;-)

File: plugins/editors/codemirror/codemirror.xml [hunk 5]
```diff
@@ -177,72 +203,102 @@
...
+					filter="options"
+					>
 					<option value="Shift">PLG_CODEMIRROR_FIELD_VALUE_FULLSCREEN_MOD_SHIFT</option>
 					<option value="Cmd">PLG_CODEMIRROR_FIELD_VALUE_FULLSCREEN_MOD_CMD</option>
 					<option value="Ctrl">PLG_CODEMIRROR_FIELD_VALUE_FULLSCREEN_MOD_CTRL</option>
 					<option value="Alt">PLG_CODEMIRROR_FIELD_VALUE_FULLSCREEN_MOD_ALT</option>
 				</field>
 
-				<field name="basePath" type="hidden" default="media/editors/codemirror/" />
-				<field name="modePath" type="hidden" default="media/editors/codemirror/mode/%N/%N" />
```
Comment (JoomliC) [inline plugins/editors/codemirror/codemirror.xml]: Maybe could stay on one line, as only a few attributes, and less 100 lenght ?

File: plugins/editors/codemirror/codemirror.xml [hunk 6]
```diff
@@ -177,72 +203,102 @@
...
+					>
 					<option value="">JDEFAULT</option>
 				</field>
 
-				<field name="activeLineColor" type="color" default="#a4c2eb" filter="color"
-					   description="PLG_CODEMIRROR_FIELD_ACTIVELINE_COLOR_DESC"
-					   label="PLG_CODEMIRROR_FIELD_ACTIVELINE_COLOR_LABEL" />
+				<field
+					name="activeLineColor"
+					type="color" default="#a4c2eb"
```
Comment (JoomliC) [inline plugins/editors/codemirror/codemirror.xml]: <code>default</code> to be moved after <code>description</code>

File: plugins/editors/codemirror/codemirror.xml [hunk 7]
```diff
@@ -177,72 +203,102 @@
...
+					description="PLG_CODEMIRROR_FIELD_ACTIVELINE_COLOR_DESC"
+					filter="color"
+				/>
 
-				<field name="highlightMatchColor" type="color" default="#fa542f" filter="color"
-					   description="PLG_CODEMIRROR_FIELD_HIGHLIGHT_MATCH_COLOR_DESC"
-					   label="PLG_CODEMIRROR_FIELD_HIGHLIGHT_MATCH_COLOR_LABEL" />
+				<field
+					name="highlightMatchColor"
+					type="color" default="#fa542f"
```
Comment (JoomliC) [inline plugins/editors/codemirror/codemirror.xml]: <code>default</code> to be moved after <code>description</code>

**Comment by andrepereiradasilva** [comment]
I have tested this item :white_check_mark: successfully on cc31a8fc587fe5e69eea7e2e1c7d7e290bb8ec9b<br /><br />on code review<hr /><sub>This comment was created with the <a href="https://github.com/joomla/jissues">J!Tracker Application</a> at <a href="https://issues.joomla.org/tracker/joomla-cms/10482">issues.joomla.org/joomla-cms/10482</a>.</sub>

**Comment by andrepereiradasilva** [comment]
I have tested this item :white_check_mark: successfully on efff868521be4dc4b0cd71b0cdb9257eb959523c<hr /><sub>This comment was created with the <a href="https://github.com/joomla/jissues">J!Tracker Application</a> at <a href="https://issues.joomla.org/tracker/joomla-cms/10482">issues.joomla.org/joomla-cms/10482</a>.</sub>

**Comment by wojsmol** [comment]
@andrepereiradasilva @JoomliC Please test

**Comment by wojsmol** [comment]
@andrepereiradasilva Thanks

**Comment by JoomliC** [comment]
I have tested this item :white_check_mark: successfully on efff868521be4dc4b0cd71b0cdb9257eb959523c<br /><br />On code review, and all is good!

@wojsmol Thanks :+1: <hr /><sub>This comment was created with the <a href="https://github.com/joomla/jissues">J!Tracker Application</a> at <a href="https://issues.joomla.org/tracker/joomla-cms/10482">issues.joomla.org/joomla-cms/10482</a>.</sub>

**Comment by brianteeman** [comment]
RTC<hr /><sub>This comment was created with the <a href="https://github.com/joomla/jissues">J!Tracker Application</a> at <a href="https://issues.joomla.org/tracker/joomla-cms/10482">issues.joomla.org/joomla-cms/10482</a>.</sub>

**Pull request merged**